.circleci/config.yml

```diff
@@ -29,3 +29,20 @@ jobs:
           # async send to zipkin, so wait a bit before reading back
           sleep 1
           test $(curl -s 127.0.0.1:9411/api/v2/traces | jq '.[0] | length') -eq 3
+
+workflows:
```

Comment:
neat. didn't know we could do this.. wonder if we can somehow build against latest by default?

Reply:
In out specific case we use the [latest commit](https://github.com/openzipkin/zipkin-php-example/blob/master/composer.json#L6) in zipkin-php and since we don't cache anything it will run against the latest version :D.